terraform/aws/variables.tf

```diff
@@ -19,7 +19,7 @@ variable "tags" {
 variable "image" {
   description = "AMI to use for the instance"
   type        = string
-  default     = "al2-chronicle*"
+  default     = "764336703387/AlmaLinux OS 8.8.2023*"
```

Comment:
Should/could this default be in hiera?

Reply:
I think putting defaults in the provider's hiera make it easier to find and maintain them.